backend/doc/apache.md

```diff
@@ -0,0 +1,93 @@
+# Running Tracim through Apache #
+
+### Installation ###
+
+Install `tracim` first.
+Install `Apache` server and uWSGI:
+
+    sudo apt install apache2 libapache2-mod-wsgi-py3 uwsgi uwsgi-plugin-python3
+
+### Configuration ###
+
+Create a file named `/etc/apache2/sites-available/tracim.conf` containing: 
+
+    <VirtualHost *:80>
+        ServerName tracim
+
+        <Directory "/">
+            Require all granted
+            Dav On
+        </Directory>
+        SetEnv proxy-sendcl
+        ProxyPreserveHost On
+
+        ProxyPass /webdav http://127.0.0.1:3030/webdav
+        ProxyPassReverse /webdav http://127.0.0.1:3030/webdav
+
+        ProxyPass /agenda http://127.0.0.1:5232/agenda
+        ProxyPassReverse /agenda http://127.0.0.1:5232/agenda
+
+        ProxyPass /ui http://127.0.0.1:6543/ui
+        ProxyPassReverse /ui http://127.0.0.1:6543/ui
+
+        ProxyPass /api http://127.0.0.1:6543/api
+        ProxyPassReverse /api http://127.0.0.1:6543/api
+
+        ProxyPass /custom_toolbox-assets http://127.0.0.1:6543/custom_toolbox-assets
+        ProxyPassReverse /custom_toolbox-assets http://127.0.0.1:6543/custom_toolbox-assets
+
+        ProxyPassMatch ^/$ http://localhost:6543
+        ProxyPassReverse ^/$ http://127.0.0.1:6543
+
+        CustomLog /var/log/apache2/apache2-tracim-access.log combined
+        ErrorLog /var/log/apache2/apache2-tracim-error.log
+
+    </VirtualHost>
+
+If you want to used browser cache policy, an exemple is visible [here](https://github.com/tracim/tracim/blob/develop/tools_docker/Debian_Uwsgi/apache2.conf.sample).
+:warning: This line `RequestHeader edit "If-None-Match" '^"((.*)-(gzip|br))"$' '"$1", "$2"'` is make to solved apache2 issue visible [here](https://bz.apache.org/bugzilla/show_bug.cgi?id=45023#c26)
+In this case you need also to make sure you used cache token feature available in development.ini
+
+Enable this configuration file:
+
+    sudo ln -s /etc/apache2/sites-available/tracim.conf /etc/apache2/sites-enabled/tracim.conf
+
+
+Create the file named `/etc/uwsgi/apps-available/tracim.ini` containing:
+
+    [uwsgi]
+    plugins = python3
```

Comment:
add virtualenv parameter, by default using setup_default_backend.sh , we need to use virtualenv (/backend/env ?), look in tracim v1 documentation.

Reply:
Done